spec/20-http_request_header_format.md

```diff
@@ -273,15 +273,12 @@ A `list-member` contains a key/value pair.
 The key is an identifier that describes the vendor.
 
 ``` abnf
-key = lcalpha 0*255( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
-key = ( lcalpha / DIGIT ) 0*240( lcalpha / DIGIT / "_" / "-"/ "*" / "/" ) "@" lcalpha 0*13( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
+key = lcalpha 0*255( keychar )
```

Comment:
You changed the semantics of the vendor key to not allow the first character to be a digit. This is important for at least the newrelic folks.

```suggestion
key = ( lcalpha / DIGIT ) 0*255 ( keychar )
```